### DISCUSSION TITLE
Does it work on windows11?

### DISCUSSION THREAD

**Discussion by CCCCCalvinCai** [coqui-ai/TTS#3732] [General]
Hi, according to installation instruction on windows. It is necessary to install CUDA Toolkit10.1 and cuDNN v7.6.5. However, I found that These will not work in windows11. My system is window11. Is that mean I cant use TTS?
![Weixin Image_20240511140432](https://github.com/coqui-ai/TTS/assets/169406522/9286ec42-1efd-4cd8-bd47-202a73e72656)

**Comment by identificator** [comment]
It's work even on latest CUDA 12.4.
For Win 11 you should see the next stackoverflow answer in this thread.
